UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamageDiscriptors.cs

```diff
@@ -23,88 +25,117 @@ public partial class BodyPart
 		private string pireceDamageDescOnLARGE= "{readableName} suffers from a Ruptured Cavity.";
 		[SerializeField] private string internalDamageDesc	 = "This {readableName} is suffering from internal damage.";
 		[SerializeField] private string externalBleedingDesc = "This {readableName} is bleeding due to an open wound.";
+		[SerializeField, NaughtyAttributes.EnableIf("CanBeBroken")] private string BoneFracturedLvlThreeDesc = "This {readableName} is suffering Hairline Fracture.";
+		[SerializeField, NaughtyAttributes.EnableIf("CanBeBroken")] private string BoneFracturedLvlTwoDesc = "This {readableName} is suffering Compound Fracture. It is completely snapped in half.";
 
 		public string GetFullBodyPartDamageDescReport()
 		{
-			string report = $"Analyizing damage report for {BodyPartReadableName}.. \n";
+			StringBuilder report = new StringBuilder();
+			report.Append($"Analyzing damage report for {BodyPartReadableName}.. \n");
 
-			report += "[Open Wound Size -> ";
+			if (CanBeBroken)
+			{
+				CheckIfBroken();
+				report.Append("[Fracture Level]\n");
+				if (isFractured_Compound)
+				{
+					report.Append(BoneFracturedLvlThreeDesc);
```

Comment:
same here. Use ``report.AppendLine()``instead of manually adding the new line in the string.